### ISSUE TITLE
Translations data is missing in getTranslations

### ISSUE THREAD

**Issue by sheriffmarley** [php-tmdb/api#188]
The translation data attribute is missing in the `translations` collection.

Tmdb provides
```
      "iso_3166_1": "NL",
      "iso_639_1": "nl",
      "name": "Nederlands",
      "english_name": "Dutch",
      "data": {
        "title": "",
        "overview": "Een verzekeringsinspecteur die aan chronische slapeloosheid lijdt probeert wanhopig uit zijn oersaaie bestaan te ontsnappen. Bij toeval ontmoet hij Tyler Durden, een charismatische zeepverkoper met een bizarre levensfilosofie. Tyler gelooft namelijk dat \"zelfverbetering\" enkel voor de zwakken is, het is \"zelfvernieling\" die het leven meer waarde geeft. Samen met Tyler organiseert hij de ultieme vorm van ontspanning: zogeheten \"Fight Clubs\", waar gewone jonge mannen het beest in zichzelf loslaten in blotevuistgevechten. Het bevrijdende effect op de deelnemende mannen doet een vreemd soort kameraadschap ontstaan, dat al snel gevaarlijke vormen aanneemt.",
        "homepage": ""
      }
```

the plugin only returns:
```
        -englishName: "Italian"
        -iso6391: "it"
        -name: "Italiano"
```

The complete data part is missing.

**Comment by anand-371** [comment]
I am interested can i be assigned

**Comment by wtfzdotnet** [comment]
@sheriffmarley this indeed is a quite big bug, that is unfortunately not isolated to just a single entity, I'll be going over everything to make sure this gets fixed everywhere.

**Comment by johannesx75** [comment]
This was fixed by #197 and is in release 2.1.17.
But Packagist has not updated yet. @wtfzdotnet can you somehow trigger an update there? https://packagist.org/packages/php-tmdb/api

**Comment by wtfzdotnet** [comment]
Did a manual update, will look into the official configuration later.
